actions.yaml

```diff
@@ -62,3 +62,5 @@ set-tls-private-key:
     private-key:
       type: string
       description: The content of private key for communications with clients. Content will be auto-generated if this option is not specified.
+reenable-oversee-users:
+  description: Reenable purging of managed credentials after a standby cluster is promoted.
```

Comment:
any reason why it is not run as part of `promote-to-primary` action?

Reply:
The original issue was the promoting the replica cluster deleted users for relations to the old primary cluster, since there were no relations for them in the new primary cluster. We can't know if the client charms got rerelated to the new primary cluster, so we can't know when it's safe to reenable that functionality without user input.